packages/network-query/queries/network/statechannel.gql

```diff
@@ -37,3 +37,98 @@ query GetStateChannels($status: ChannelStatus!) {
     }
   }
 }
+
+query GetExpiredStateChannels($now: Datetime!) {
+  stateChannels(filter: { status: { equalTo: OPEN }, expiredAt: { lessThanOrEqualTo: $now } }) {
+    nodes {
+      ...StateChannelFields
+    }
+  }
+}
+
+# CONSUMER
+
+query GetConsumerOngoingFlexPlans($consumer: String!, $now: Datetime!, $offset: Int) {
+  stateChannels(
+    filter: {
+      consumer: { equalTo: $consumer }
+      expiredAt: { greaterThan: $now }
+      status: { equalTo: OPEN }
+    }
+    offset: $offset
+  ) {
+    totalCount
+    nodes {
+      ...StateChannelFields
+    }
+  }
+}
+
+query GetConsumerClosedFlexPlans($consumer: String!, $now: Datetime!, $offset: Int) {
```

Comment:
Similar with `GetIndexerClosedFlexPlans`.
Put `consumer: { equalTo: $consumer }` out of `or` operation.
closed_plans = finalised_plan + expired_plans